.github/workflows/image_release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Image Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "The Operator branch/tag to checkout when creating the image."
+        required: true
+        default: "release_operator_image"
+      tags:
```

Comment:
Do you have a specific scenario in mind where we would need separate input for `ref` and `tags`? I think a ref alone should suffice as this can reference a branch or tag.